src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -145,18 +148,7 @@ public Object createModule(String name) throws CheckstyleException {
         // if the name is a simple class name, try to find it in maps at first
         if (!name.contains(PACKAGE_SEPARATOR)) {
             // find the name in hardcode map
-            final String fullModuleName = NAME_TO_FULL_MODULE_NAME.get(name);
-            if (fullModuleName == null) {
-                final String fullCheckModuleName =
-                        NAME_TO_FULL_MODULE_NAME.get(name + CHECK_SUFFIX);
-                if (fullCheckModuleName != null) {
-                    instance = createObject(fullCheckModuleName);
-                }
-            }
-            else {
-                instance = createObject(fullModuleName);
-            }
-
+            instance = createFromExistingName(name);
```

Comment:
>
```
// find the name in hardcode map
instance = createFromExistingName(name);
```

comments are to explain code, comments for method - is sign of bad name design of method. 
`createFromStandardCheckSet` ?

Reply:
method renamed and comment deleted